pkg/transforms/genericResourceConfig.go

```diff
@@ -126,8 +126,17 @@ var defaultTransformConfig = map[string]ResourceConfig{
 		properties: []ExtractProperty{
 			{Name: "ready", JSONPath: `{.status.conditions[?(@.type=='Ready')].status}`},
 			{Name: "status", JSONPath: `{.status.conditions[?(@.type=='Ready')].reason}`},
+			{Name: "phase",  JSONPath: `{.status.phase}`},
 			{Name: "indications", JSONPath: `{.status.indications}`}, // this is an array of strings - will collect array items separated by ;
-			{Name: "sourceVM", JSONPath: `{.spec.source.name}`},
+			{Name: "_sourceVM", JSONPath: `{.spec.source.name}`},
```

Comment:
Let's match the CLI here too.

<img width="993" alt="image" src="https://github.com/user-attachments/assets/13b5574c-7b9f-4387-ad9e-094136cc74c7" />